src/components/SlidePanel/_slide-panel.scss

```diff
@@ -11,6 +11,12 @@
   position: fixed;
   right: 0;
   top: $top-header-height;
+  transform: translateX(100%);
+  transition: transform 0.5s;
```

Comment:
This feels a bit slow, can we speed it up a bit?

Reply:
Please use Vanilla animation speeds please.